app/dashboard/templates/yge/send2.html

```diff
@@ -1,7 +1,9 @@
 {% extends 'yge/base.html' %}
-
+{% load staticfiles %}
 {% block 'scripts' %}
 
+{#<script src="{% static 'v2/js/jquery.js' %}"></script>#}

```

Comment:
should this be commented out?

Reply:
yes, it is commented out intentionally. I will probably remove it out in the next commit